rubygem-foreman_templates/rubygem-foreman_templates.spec

```diff
@@ -35,6 +35,8 @@ Requires: %{?scl_prefix}rubygem(git)
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: foreman-plugin >= 1.15
```

Comment:
It is needed now, don't remove it. The apipie docs (pre)compilation requires it, I was just saying that it is related to apipie docs.